scripts/utilities/rescal_utilities.py

```diff
@@ -5,381 +5,474 @@
-
-def __default_options(self):
-    # set default options
-    # ReSCAL lists a lot of command-line options in entry.c, show_general_options()
-    self.options['clean'] 		= True
-    self.options['backup'] 		= False
-    self.options['parfile'] 	= "sno_cone.par"
-    self.options['genesislog']	= "GENESIS.log"
-    self.options['rescallog']	= "RESCAL.log"
-    self.options['rescallocation']	= "../src"
-
-    # automatically handle command line options here
-    self.__set_flag_option('usage info',		'h',	True)
-    self.__set_flag_option('show params', 		'hm', 	True)
-    self.__set_flag_option('no video',		'nv',	True)
-    self.__set_flag_option('info interval', 	'info',	True)
-    self.__set_flag_option('output interval', 	'dcsp', '10t0')
-    self.__set_flag_option('png interval',		'dpng', '10t0')
-    self.__set_flag_option('stop after',		'stop', False)
-    self.__set_flag_option('frame rate', 		'fr',	False)
-    self.__set_flag_option('random seed',		's',	False)
-    self.__set_flag_option('vel', 			'vel', 	True)
-    self.__set_flag_option('vss', 			'vss', 	True)
+    # Creates a ReSCAL run script
+    # Yes, it's a python script that writes a bash script, and yes, that's a bit silly
+    def __init__(self):
+        self.options = {}
+        self.__flag_options = {}
+        self.__default_options()
+        self.__default_header()
+
+    def __default_header(self):
+        self.header_start = "Script written with Run_Script utility (KK)"
+        self.header_body  = "Default options"
+
+    def __default_options(self):
+        # set default options
+        # ReSCAL lists a lot of command-line options in entry.c, show_general_options()
+        self.options['clean']           = True
+        self.options['backup']          = False
+        self.options['parfile']         = "sno_cone.par"
+        self.options['genesislog']      = "GENESIS.log"
+        self.options['rescallog']       = "RESCAL.log"
+        self.options['rescallocation']  = "../src"
+
+        # automatically handle command line options here
+        self.__set_flag_option('usage info',            'h',    True)
+        self.__set_flag_option('show params',           'hm',   True)
+        self.__set_flag_option('no video',              'nv',   True)
+        self.__set_flag_option('info interval',         'info', True)
+        self.__set_flag_option('output interval',       'dcsp', '10t0')
+        self.__set_flag_option('png interval',          'dpng', '10t0')
+        self.__set_flag_option('stop after',            'stop', False)
+        self.__set_flag_option('frame rate',            'fr',   False)
+        self.__set_flag_option('random seed',           's',    False)
+        self.__set_flag_option('vel',                   'vel',  True)
+        self.__set_flag_option('vss',                   'vss',  True)
+        self.__set_flag_option('quit',                  'q',   False)
+        self.__set_flag_option('alti only',             'altionly', False)
```

Comment:
I've added an option to use the preexisting quit flag and also added the new altionly flag